### PR TITLE
Setup the ability to select single vs bookview

### PR DESCRIPTION
**Pull request by mejackreed**
Part of #1790 

bookviewselection

This wires up the UI components for selecting the different view types.

I split this out in to its own PR, as I think implementing #1790 in a single PR is going to be quite large.

### CODE REVIEW

**Comment by mejackreed** [comment]
I'm using `view` right now as it directly relates to the user interface element in the designs, rather than a direct translation from the specs. It definitely seems like there will be some more work needed to support how `viewingHint` and `behavior` works here, but I'm considering that out of scope for this work as outlined in #1790.

File: __tests__/src/selectors/index.test.js [hunk 1]
```diff
@@ -96,6 +97,7 @@ describe('getThumbnailNavigationPosition', () => {
   });
 });
 
+<<<<<<< HEAD
```
Comment (cbeer) [inline __tests__/src/selectors/index.test.js]: Merge conflict?

Comment (mejackreed) [reply __tests__/src/selectors/index.test.js]: ah.. nice catch :)

File: src/state/actions/window.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ export function addWindow(options) {
     thumbnailNavigationPosition: 'bottom', // bottom by default in settings.js
     xywh: [0, 0, 400, 400],
     rotation: null,
+    view: 'single',
```
Comment (camillevilla) [inline src/state/actions/window.js]:
```suggestion
    viewType: 'single',
```
This is a small suggestion, but we might save ourselves a tiny headache down the line by having this config key match our action types name.

Comment (mejackreed) [reply src/state/actions/window.js]: If its ok, can we hold off on this name change? I agree its suboptimal right now. Perhaps we can brainstorm to figure out something better for internal/and external usage?

**Pull request merged**